src/templates/static/new_layout/style_guide.jsx

```diff
@@ -92,6 +93,39 @@ const StyleGuide = () => (
                         />
                     </div>
                 </div>
+                <div className='gr-row gr-padding-20'>
+                    <div className='gr-12'>
+                        <h2 className='center-text'>Tooltips</h2>
+                        <div className='gr-row'>
+                            <div className='gr-6'>
+                                <p className='center-text'>
+                                    <Tooltip alignment='top' message='Helper text here.' />
+                                    <span style={{ fontSize: '12px' }}>Top</span>
```

Comment:
How can we avoid inline styles?

Reply:
I think we can ignore it for now in this page, as this style-guide is just temporary and we should replace it with a style-guide page that shows both code and preview (similar to binary-style) so we can also include static components which are stateful like `Tabs` and `SideMenu`.

Reply:
Sure, thanks.